db/schema.rb

```diff
@@ -442,6 +442,16 @@
     t.index ["file_name"], name: "index_data_update_scripts_on_file_name", unique: true
   end
 
+  create_table "devices", force: :cascade do |t|
+    t.string "app_bundle", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.string "platform", null: false
+    t.string "token", null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.bigint "user_id", null: false
+    t.index ["user_id", "token", "platform", "app_bundle"], name: "index_devices_on_user_id_and_token_and_platform_and_app_bundle", unique: true
+  end
+
```

Comment:
This is the only `schema.rb` change that needs to be included. 

The other changes (including `rpush` migrations) were run while I was investigating this and slipped in. I'll get rid of them in following commits

Reply:
✅ Removed those lingering `rpush` table migrations!